sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3210,17 +3230,15 @@ class _DeferredGroupByCols(frame_base.DeferredFrame):
   diff = frame_base._elementwise_method('diff', base=DataFrameGroupBy)
   fillna = frame_base._elementwise_method('fillna', base=DataFrameGroupBy)
   filter = frame_base._elementwise_method('filter', base=DataFrameGroupBy)
-  first = frame_base.wont_implement_method(
-      DataFrameGroupBy, 'first', reason="order-sensitive")
+  first = frame_base._elementwise_method('first', base=DataFrameGroupBy)
```

Comment:
Putting it here cause it to do a `first` globally?

Reply:
This is a weird quirk of our implementation. In pandas when you groupby() a DataFrame you can change the "axis" you want to group/aggregate across. The default is the intuitive axis="index", where each column is grouped/aggregated across all of the rows of the dataset.

But users can also specify they want to groupby(axis="columns"), in which case each _row_ will be grouped/aggregated across the columns. This class, `_DeferredGroupByCols`. is just handling that `axis="columns"` case.

Technically we can easily support most of these aggregations since they're just performing an operation on each element, but it's not clear this path actually gets much usage.